### PR TITLE
Read the shouldHideReaderRevenue flag

### PR DESCRIPTION
**Pull request by jranks123**
## What does this change?
A new flag which is to be read by our acquisition asks, shouldHideReaderRevenue, is how available in CAPI (https://github.com/guardian/content-api/pull/1951).

This PR adds the funcitonalilty to read this value and make it available as part of window.guardian.config.page

## What is the value of this and can you measure success?
It means we can have greater control over what content we feel comfortable showing acquisition asks on.

## Does this affect other platforms - Amp, Apps, etc?
No

## Screenshots

screenshot at jun 15 09-50-20

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
@jranks123 Please note that this field will only be present on content since it was released. Therefore for all content prior to it's release you will need to still handle it as you used to. Thanks.

File: common/app/model/content.scala [hunk 1]
```diff
@@ -424,6 +424,7 @@ object Article {
       ("isHosted", JsBoolean(false)),
       ("isPhotoEssay", JsBoolean(content.isPhotoEssay)),
       ("isSensitive", JsBoolean(fields.sensitive.getOrElse(false))),
+        ("shouldHideReaderRevenue", JsBoolean(fields.shouldHideReaderRevenue)),
```
Comment (NataliaLKB) [inline common/app/model/content.scala]: Nit: might be just me, but the indentation here looks off.

**Comment by jranks123** [comment]
@LATaylor-guardian Correct, this PR doesn't deal with how we use the flag, that's coming next, and we are going to obey the old rules before a cut-off date, then respect this new flag thereafter

File: common/app/model/content.scala (hunk 1 above)
Comment (jranks123) [inline common/app/model/content.scala]: Ah yes, good spot

**Review by NataliaLKB — APPROVED** [review]
Code changes look good. Lots of tests failing though!

**Pull request merged**